packages/cozy-konnector-libs/src/libs/categorization/localModel/classifier.js

```diff
@@ -36,8 +36,7 @@ const TOKENS_TO_REWEIGHT = [
   'tag_pos tag_activity_income'
 ]
 
-const getClassifierOptions = transactionsWithManualCat => {
-  const uniqueCategories = getUniqueCategories(transactionsWithManualCat)
+const getClassifierOptions = uniqueCategories => {
```

Comment:
By reading the code, I think you can just pass the length of the categories rather than the actual array.
Also, could you add some jsdoc? Like: 
```
/**
 * Get the classifier options, mainly to get the alpha parameter 
 *
 * @param {number} nUniqueCategories - Number of unique categories
 * @returns {object} the classifier options
 */
 ```